### ISSUE TITLE
Cannot start REPL with s3-wagon-private Leiningen plugin on Cursive 0.1.64

### ISSUE THREAD

**Issue by mkorkeal** [cursiveclojure/cursive#1115]
After upgrade from 0.1.63 to 0.1.64 the REPL fails to start with exception. Reverting back to 0.1.63 fixes the issue.
project.clj contains following:

```
:plugins [[s3-wagon-private "1.1.2"]
            ;; Pass environment config from project.clj to the application
            [lein-environ "1.0.1"]]
```

Logs:

```
2015-10-22 10:36:03,305 [  24140]  ERROR - plication.impl.ApplicationImpl - java.lang.ClassNotFoundException: org.springframework.aws.maven.PrivateS3Wagon, compiling:(NO_SOURCE_PATH:0:0) 
java.lang.ClassNotFoundException: org.springframework.aws.maven.PrivateS3Wagon, compiling:(NO_SOURCE_PATH:0:0)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:6730)
        at clojure.lang.Compiler.analyze(Compiler.java:6524)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:6711)
        at clojure.lang.Compiler.analyze(Compiler.java:6524)
        at clojure.lang.Compiler.analyze(Compiler.java:6485)
        at clojure.lang.Compiler$BodyExpr$Parser.parse(Compiler.java:5861)
        at clojure.lang.Compiler$FnMethod.parse(Compiler.java:5296)
        at clojure.lang.Compiler$FnExpr.parse(Compiler.java:3925)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:6721)
        at clojure.lang.Compiler.analyze(Compiler.java:6524)
        at clojure.lang.Compiler.analyze(Compiler.java:6485)
        at clojure.lang.Compiler$BodyExpr$Parser.parse(Compiler.java:5861)
        at clojure.lang.Compiler$FnMethod.parse(Compiler.java:5296)
        at clojure.lang.Compiler$FnExpr.parse(Compiler.java:3925)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:6721)
        at clojure.lang.Compiler.analyze(Compiler.java:6524)
        at clojure.lang.Compiler.eval(Compiler.java:6779)
        at clojure.lang.Compiler.eval(Compiler.java:6745)
        at clojure.core$eval.invoke(core.clj:3081)
        at leiningen.core.project$load_plugins.invoke(project.clj:700)
        at leiningen.core.project$load_plugins.invoke(project.clj:703)
        at leiningen.core.project$set_profiles.doInvoke(project.clj:846)
        at clojure.lang.RestFn.invoke(RestFn.java:442)
        at leiningen.core.project$merge_profiles.invoke(project.clj:855)
        at leiningen.repl$repl.doInvoke(repl.clj:310)
        at clojure.lang.RestFn.invoke(RestFn.java:425)
        at cursive.leiningen.repl$repl_details$fn__1324.invoke(repl.clj:60)
        at cursive.leiningen.repl$repl_details.invoke(repl.clj:58)
        at clojure.lang.Var.invoke(Var.java:379)
        at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:88)
        at org.projectodd.shimdandy.impl.ClojureRuntimeShimImpl.invoke(ClojureRuntimeShimImpl.java:81)
        at cursive.leiningen.ShimBootstrap$1.invoke(ShimBootstrap.java:36)
        at cursive.repl.process_runner$create_lein_parameters.invoke(process_runner.clj:302)
        at cursive.repl.process_runner$create_run_profile_state$reify__12318.createJavaParameters(process_runner.clj:339)
        at com.intellij.execution.configurations.JavaCommandLineState.getJavaParameters(JavaCommandLineState.java:35)
        at cursive.repl.runner.LocalReplRunner$2.run(LocalReplRunner.java:145)
        at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:563)
        at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:152)
        at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:452)
        at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:402)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
        at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:137)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl$1.run(ProgressManagerImpl.java:126)
        at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:400)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
        at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
Caused by: java.lang.ClassNotFoundException: org.springframework.aws.maven.PrivateS3Wagon
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:348)
        at clojure.lang.RT.classForName(RT.java:2154)
        at clojure.lang.RT.classForName(RT.java:2163)
        at clojure.lang.Compiler$HostExpr.maybeClass(Compiler.java:1017)
        at clojure.lang.Compiler$HostExpr.access$600(Compiler.java:795)
        at clojure.lang.Compiler$NewExpr$Parser.parse(Compiler.java:2609)
        at clojure.lang.Compiler.analyzeSeq(Compiler.java:6723)
        ... 49 more2015-10-22 10:36:03,307 [  24142]  ERROR - plication.impl.ApplicationImpl - IntelliJ IDEA 14.1.5  Build #IU-141.2735.5 
2015-10-22 10:36:03,307 [  24142]  ERROR - plication.impl.ApplicationImpl - JDK: 1.8.0_60 
2015-10-22 10:36:03,307 [  24142]  ERROR - plication.impl.ApplicationImpl - VM: OpenJDK 64-Bit Server VM 
2015-10-22 10:36:03,307 [  24142]  ERROR - plication.impl.ApplicationImpl - Vendor: Oracle Corporation 
2015-10-22 10:36:03,307 [  24142]  ERROR - plication.impl.ApplicationImpl - OS: Linux 
2015-10-22 10:36:03,307 [  24142]  ERROR - plication.impl.ApplicationImpl - Last Action: Run 
```

**Comment by danielcompton** [comment]
I'm seeing this too, though it didn't happen on earlier beta test versions of Cursive.

**Comment by cursiveclojure** [comment]
@danielcompton reported that this issue is fixed in 0.1.65, so I'm going to close this. Feel free to re-open if it recurs.

**Comment by da-z** [comment]
This happens in the latest build as well. To reproduce, try to download the sources for a plugin from within IntelliJ (by clicking "Download sources" in the pop-up).
